src/Utility/Composer.php

```diff
@@ -121,7 +121,7 @@ public static function getOAUTHToken(CommandRunner $runner)
         // try composer stored oauth token
         $command = ['composer', 'config', '-g', 'github-oauth.github.com'];
         $error = "Couldn't determine GitHub oAuth token. Please set GITHUB_API_TOKEN";
-        $result = $runner->runCommand($command, $error);
+        $result = $runner->runCommand($command, $error, true, false);
```

Comment:
```suggestion
        $result = $runner->runCommand($command, $error, allowDebugVerbosity: false);
```
Instead of passing in a default value, we can just skip that argument.
I'm not fussed if you ignore this suggestion - it seems a little tidier to me but it's not a deal breaker either way.

Reply:
I really dislike this suggestion since it's mixing named with unnamed arguments